bundles/framework/featuredata/plugin/FeatureDataPlugin.js

```diff
@@ -92,15 +92,16 @@ Oskari.clazz.define('Oskari.mapframework.bundle.featuredata.plugin.FeatureDataPl
                 <ThemeProvider value={this.getMapModule().getMapTheme()}>
                     <MapModuleTextButton
                         visible={layers?.length > 0}
-                        text={<Message messageKey='title' bundleKey='FeatureData'/>}
                         onClick={() => this.handler.openFlyout()}
                         active={flyoutOpen}
                         loading={loadingStatus.loading}
                         position={this.getLocation()}
                         $marginRight={marginRight}
                         $marginLeft={marginLeft}
                         $marginTop={'10'}
-                    />
+                    >
+                        {<Message messageKey='title' bundleKey='FeatureData'/>}
```

Comment:
Should work without the {}?
```suggestion
                        <Message messageKey='title' bundleKey='FeatureData'/>
```